components/brave_wallet_ui/stories/locale.ts

```diff
@@ -591,6 +600,8 @@ provideStrings({
   braveWalletSwapReviewHeader: 'Confirm order',
   braveWalletSolanaSwap: 'Solana Swap',
   braveWalletNoRoutesFound: 'No routes found',
+  braveWalletSwappingVia: 'Swapping via',
```

Comment:
should this use replacements?
```suggestion
  braveWalletSwappingVia: 'Swapping via $1',
```

Reply:
Is not used in the same `string`.

![Screenshot 12](https://github.com/brave/brave-core/assets/40611140/e15c7999-95d7-4a82-bb24-4f6277154d49)